<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="BO" failures="1" tests="6" name="ExelenterTestNgProject" time="91.796" errors="0" timestamp="2023-02-16T20:36:04 EST">
  <testcase classname="com.exelenter.class04._06HW" name="addEmployees" time="5.837"/>
  <testcase classname="com.exelenter.class04._06HW" name="addEmployees" time="5.238"/>
  <testcase classname="com.exelenter.class04._06HW" name="addEmployees" time="5.538"/>
  <testcase classname="com.exelenter.class04._06HW" name="addEmployees" time="5.501"/>
  <testcase classname="com.exelenter.class04._06HW" name="addEmployees" time="40.464">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.exelenter.class04._06HW.addEmployees(_06HW.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- addEmployees -->
  <testcase classname="com.exelenter.class04._06HW" name="addEmployees" time="5.402"/>
  <testcase classname="com.exelenter.class04._06HW" name="deleteUsers" time="0.0">
    <ignored/>
  </testcase> <!-- deleteUsers -->
</testsuite> <!-- ExelenterTestNgProject -->
